packages/remark-codesandbox/src/index.js

```diff
@@ -101,10 +101,19 @@ function codesandbox(options = {}) {
         );
       }
 
+      const overrideEntry =
+        query.get('overrideEntry') !== 'false' ||
+        options.overrideEntry ||
```

Comment:
I think we don't have to make this option available in the top most config, one can just pass it to `query` to set it globally. Less configuration surface.

```js
query: {
  overrideEntry: false,
}
```

Reply:
Ok, no problem. I didn't want to assume I could pollute the `options.query` object.

But if you're fine with it, going ahead with this!

Reply:
Not polluting the `options.query` object, `query` should already includes the key specified in `options.query`. See `baseQuery`.